assets/airports/essa.json

```diff
@@ -0,0 +1,3558 @@
+        "KOGAV":        ["N60d04m52.00", "E017d13m46.60"],
+        "LUMAX":        ["N60d19m03.80", "E018d00m30.80"],
+        "MENGA":        ["N59d25m51.40", "E018d36m40.50"],
+        "NEKLA":        ["N59d00m00.00", "E019d15m49.10"],
+        "NILUG":        ["N58d48m57.00", "E017d53m05.00"],
+        "NOSLI":        ["N59d04m22.00", "E017d15m29.20"],
+        "OGMAK":        ["N59d22m49.50", "E017d59m40.60"],
+        "OXINU":        ["N59d50m06.60", "E017d59m06.60"],
+        "PAZIX":        ["N59d36m49.90", "E017d35m26.40"],
+        "PETEV":        ["N59d12m25.80", "E017d00m43.50"],
+        "RESNA":        ["N60d22m01.00", "E018d01m29.40"],
+        "REYMY":        ["N59d36m03.30", "E018d17m26.00"],
+        "ROKNI":        ["N59d35m03.90", "E016d44m55.20"],
+        "TALEK":        ["N60d02m54.70", "E017d18m18.50"],
+        "TOMBU":        ["N59d13m46.00", "E019d34m04.20"],
+        "TOVRI":        ["N59d44m59.30", "E018d46m00.60"],
+        "XILAN":        ["N59d39m33.50", "E019d04m33.80"],
+        "ANE":          ["N59d41m38.30", "E018d03m35.50"],
+        "ANW":          ["N59d42m47.80", "E017d51m09.30"],
+        "ARL":          ["N59d39m12.30", "E017d54m52.10"],
+        "ARS":          ["N59d35m10.40", "E016d39m01.20"],
+        "ASE":          ["N59d38m13.90", "E017d57m26.50"],
+        "ASW":          ["N59d35m15.70", "E017d49m10.90"],
+        "CM":           ["N59d54m16.20", "E017d36m02.10"],
+        "COR":          ["N59d15m41.00", "E017d29m02.70"],
+        "ERK":          ["N59d53m46.50", "E018d20m12.90"],
+        "ESA":          ["N59d39m51.30", "E017d58m20.90"],
+        "HMR":          ["N60d16m45.20", "E018d23m29.60"],
+        "LE":           ["N59d39m46.90", "E016d39m48.70"],
+        "LNA":          ["N59d32m20.30", "E017d21m30.10"],
+        "LX":           ["N59d16m31.70", "E016d42m47.30"],
+        "NSA":          ["N59d39m49.10", "E017d55m11.70"],
+        "NW":           ["N58d47m03.80", "E016d46m46.40"],
+        "ON":           ["N58d35m04.00", "E016d22m55.30"],
+        "PEO":          ["N58d47m45.90", "E017d02m39.10"],
+        "RD":           ["N59d31m03.20", "E016d36m16.30"],
+        "SB":           ["N59d21m22.80", "E017d55m58.80"],
+        "SBA":          ["N59d21m02.50", "E017d56m58.80"],
+        "SCD":          ["N59d35m54.80", "E016d38m07.20"],
+        "SP":           ["N58d35m05.00", "E016d15m48.60"],
+        "SSA":          ["N59d38m25.90", "E017d54m41.90"],
+        "SU":           ["N59d20m41.80", "E016d42m40.30"],
+        "TEB":          ["N59d31m54.10", "E018d12m11.90"],
+        "TRS":          ["N58d56m16.50", "E017d30m08.30"],
+        "TSA":          ["N59d37m44.60", "E017d57m14.80"],
+        "UP":           ["N59d58m46.20", "E017d41m46.00"],
+        "USA":          ["N59d38m44.40", "E017d57m36.40"],
+        "WA":           ["N59d39m20.80", "E017d54m55.50"],
+        "WEK":          ["N58d47m16.40", "E016d53m14.10"],
+        "ABAMA":        ["N57d59m12.00", "E015d34m11.00"],
```

Comment:
In my opinion there are too many fixes far away outside the airspace. That's since I have included the relevant airways all the way to the border of Sweden FIR (and therefore need the fixes). Later on I'm thinking of removing all fixes outside a certain distance from the airport and shortening the airways."